lib/icingadb/icingadb-objects.cpp

```diff
@@ -572,6 +578,36 @@ std::vector<String> IcingaDB::GetTypeOverwriteKeys(const String& type)
 	return std::move(keys);
 }
 
+std::vector<String> IcingaDB::GetTypeDumpSignalKeys(const String& type)
+{
+	std::vector<String> keys = {
+			m_PrefixConfigObject + type,
+			m_PrefixConfigObject + type + ":customvar",
```

Comment:
This sends this signal for objects that don't have custom vars, doesn't it? Won't break anything but all other sub-keys are special-cases already anyways.

Reply:
That's right. But actually the only type in here that doesn't have custom vars are zones. We could exclude them.

Reply:
What about comments, downtimes, and endpoints?

Reply:
Whoops, didn't think far enough with that. There's more types then just what's checked inside this function... 😆 

Yep, we should change this.